tst/com/amazon/corretto/crypto/provider/test/SecurityPropertyTester.java

```diff
@@ -31,12 +31,22 @@ public static void main(String[] args) throws Exception {
 
     String infoStr = NATIVE_PROVIDER.getInfo();
     System.out.println("Security Provider : " + infoStr);
-    assertTrue(infoStr.startsWith(NATIVE_PROVIDER.getName()));
+    String[] tokens = infoStr.split("[()]");
+    String[] semver = tokens[0].split("[\\s+-]");
+    String[] addInfo = tokens[1].split(",");
```

Comment:
Suggest not being this strict with the testing. This is supposed to just be an informative string, and we don't need to rigorously validate the format and contents. Just makes it more of a hassle to adapt it later on.